src/main/java/com/salesforce/mirus/MirusSourceTask.java

```diff
@@ -110,6 +111,12 @@ public void stop() {
     }
   }
 
+  protected void closeConsumer() {
```

Comment:
I think this should be called `shutDownTask()`, as this is the natural place for any clean-up activity that needs to be done after the last poll completes during a shutdown operation.  Closing the consumer just happens to be the only action right now.